src/core/Akka.Remote.Tests/RemotingSpec.cs

```diff
@@ -651,6 +652,21 @@ await EventFilter.Exception<OversizedPayloadException>(start: "Discarding oversi
                 });
         }
 
+        /// <summary>
+        /// Validate that we can accurately log wrapped messages that fail to be delivered
+        /// </summary>
+        [Fact]
+        public void Log_Wrapped_messages_that_fail_to_Send()
+        {
+            // 2x wrapped message
+            var wrappedMessage =
+                new DeadLetter(new ActorSelectionMessage("hit", Array.Empty<SelectionPathElement>(), false), TestActor,
+                    TestActor);
+
+            var loggedType = EndpointWriter.LogPossiblyWrappedMessageType(wrappedMessage);
+            loggedType.Should().Contain("DeadLetter").And.Contain("ActorSelectionMessage").And.Contain("String");
```

Comment:
Simple unit test to assert that our log formatting helper captures all of the layers correctly.

This test produces the following output in the log:

```
Akka.Event.DeadLetter-->(Akka.Actor.ActorSelectionMessage-->System.String)
```